#!/usr/bin/env python
PACKAGE = "cam_trigger"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Frequenz",    double_t,    0, "Hz", 30,  0, 100)
gen.add("Pulsweite", double_t, 0, "%",    20, 0,   100)
gen.add("System",   bool_t,   0, "bool",  True)
gen.add("cam1",   bool_t,   0, "bool",  True)
gen.add("cam2",   bool_t,   0, "bool",  True)
gen.add("cam3",   bool_t,   0, "bool",  True)
gen.add("cam4",   bool_t,   0, "bool",  True)

exit(gen.generate(PACKAGE, "cam_trigger", "cam_trigger"))
